pipeline/lib/membrane/video_compositor/pipeline/utility/options.ex

```diff
@@ -6,46 +6,46 @@ defmodule Membrane.VideoCompositor.Pipeline.Utils.Options do
   @typedoc """
   Specifications of the input video sources
   """
-  @type inputs_t :: [InputStream.t()]
+  @type inputs :: [InputStream.t()]
 
   @typedoc """
   Specifications of the sink element or path to the output video file
   """
-  @type output_t :: String.t() | Membrane.Sink
+  @type output :: String.t() | Membrane.Sink
 
   @typedoc """
   Specification of the output video, parameters of the final \"canvas\"
   """
-  @type stream_format_t :: RawVideo.t()
+  @type stream_format :: RawVideo.t()
 
   @typedoc """
   Multiple Frames Compositor
   """
-  @type compositor_t :: Membrane.Filter.t()
+  @type compositor :: Membrane.Filter.t()
 
   @typedoc """
   Decoder for the input buffers. Frames are passed by if `nil` given.
   """
-  @type decoder_t :: Membrane.Filter.t() | nil
+  @type decoder :: Membrane.Filter.t() | nil
 
   @typedoc """
   Encoder for the output buffers. Frames are passed by if `nil` given.
   """
-  @type encoder_t :: Membrane.Filter.t() | nil
+  @type encoder :: Membrane.Filter.t() | nil
 
   @typedoc """
   An additional plugin that sits between the decoder (or source if there is no decoder) and the compositor.
   """
-  @type input_filter_t :: Membrane.Filter.t() | nil
+  @type input_filter :: Membrane.Filter.t() | nil
 
   @type t() :: %__MODULE__{
-          inputs: inputs_t(),
-          output: output_t(),
-          stream_format: stream_format_t(),
-          compositor: compositor_t(),
-          decoder: decoder_t(),
-          encoder: encoder_t(),
-          input_filter: input_filter_t()
+          inputs: inputs(),
+          output: output(),
+          stream_format: stream_format(),
+          compositor: compositor(),
+          decoder: decoder(),
+          encoder: encoder(),
+          input_filter: input_filter()
```

Comment:
All those `something_t` -> `something` type name changes IMO are not relevant and outside of the scope of this PR. Those changes have nothing to do with scene API design.

Reply:
Ok